internal/pkg/admission/policy-server-service_test.go

```diff
@@ -1,28 +0,0 @@
-package admission
-
-import (
-	"testing"
-
-	"github.com/kubewarden/kubewarden-controller/internal/pkg/constants"
-	policiesv1 "github.com/kubewarden/kubewarden-controller/pkg/apis/policies/v1"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-)
-
-func TestMetricsEnabled(t *testing.T) {
```

Comment:
I think we should keep the metrics test around.

Reply:
Yes, I'm aware of this:

> NOTE: I've not added test to cover the metrics configurations yet. I'll add a future PR to cover that. I do not want to wait more in this PR to get some feedback from the team.

Reply:
@viccuad I've added unit test to cover the metrics configuration in this PR. No need for future PR. No need to create convoluted integration tests.  It's easier to test this with unit test here. Please, take another look